crmsh/cibquery.py

```diff
@@ -33,6 +33,27 @@ def has_primitive_filesystem_with_fstype(cib: lxml.etree.Element, fstype: str) -
         f'/instance_attributes/nvpair[@name="fstype" and @value="{fstype}"]'
     ))
 
+
+def has_primitive(cib: lxml.etree.Element, ra: ResourceAgent) -> list[str]:
```

Comment:
`has_primitive` sounds like to return a boolean.

```suggestion
def get_primitives_with_ra(cib: lxml.etree.Element, ra: ResourceAgent) -> list[str]:
```